modules/web/web-vhost/src/test/java/com/enonic/xp/web/vhost/impl/VirtualHostInternalHelperTest.java

```diff
@@ -13,6 +13,19 @@
 public class VirtualHostInternalHelperTest
 {
 
+    @Test
+    public void testNormalizePath()
+    {
+        final HttpServletRequest req = mock( HttpServletRequest.class );
+        when( req.getRequestURI() ).thenReturn( "/a/other/service" );
+
+        assertEquals( "/", VirtualHostInternalHelper.normalizePath( null ) );
+        assertEquals( "/", VirtualHostInternalHelper.normalizePath( "" ) );
+        assertEquals( "/", VirtualHostInternalHelper.normalizePath( "/" ) );
+        assertEquals( "/a", VirtualHostInternalHelper.normalizePath( "/ a /" ) );
```

Comment:
that would be suspicious if path contains raw spaces. Maybe the idea was to trim? (because value comes from config)